### ISSUE TITLE
date.timezone is referenced within config.yaml but not being set in php.ini file.

### ISSUE THREAD

**Issue by timmigrant** [puphpet/puphpet#1426]
Hi,

I can see an impact to the php.ini file through my changes to enable errors , however, as set within the config.yaml as so:

```
php:
    install: '1'
    version: '54'
    composer: '0'
    composer_home: ''
    modules:
        php:
            - cli
            - intl
            - mcrypt
            - gd
            - mbstring
        pear: {  }
        pecl:
            - pecl_http
    ini:
        display_errors: 'On'
        session.save_path: '/var/lib/php/session'
        engine: On
        short_open_tag: Off
        asp_tags: Off
        precision: 14
        y2k_compliance: On
        output_buffering: 4096
        zlib.output_compression: Off
        implicit_flush: Off
        serialize_precision: 100
        allow_call_time_pass_reference: Off
        safe_mode: Off
        safe_mode_gid: Off
        safe_mode_allowed_env_vars: 'PHP_'
        safe_mode_protected_env_vars: 'LD_LIBRARY_PATH'
        expose_php: On
        max_execution_time: 30     
        max_input_time: 60
        memory_limit: '128M'
        error_reporting: 'E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED'
        display_startup_errors: On
        log_errors: On
        log_errors_max_len: 1024
        ignore_repeated_errors: Off
        ignore_repeated_source: Off
        report_memleaks: On
        track_errors: On
        html_errors: On
        variables_order: "GPCS"
        request_order: "GP"
        register_globals: Off
        register_long_arrays: Off
        register_argc_argv: Off
        auto_globals_jit: On
        post_max_size: '8M'
        magic_quotes_gpc: Off
        magic_quotes_runtime: Off
        magic_quotes_sybase: Off
        default_mimetype: "text/html"
        enable_dl: Off
        file_uploads: On
        upload_max_filesize: '2M'
        allow_url_fopen: On
        allow_url_include: Off
        default_socket_timeout: 60
        date.timezone: 'America/Toronto'
        define_syslog_variables : Off
        SMTP: 'localhost'
        smtp_port: 25
        sendmail_path: '/usr/sbin/sendmail -t -i'
        mail.add_x_header: On
        sql.safe_mode: Off
        odbc.allow_persistent: On
        odbc.check_persistent: On
        odbc.max_persistent: '-1'
        odbc.max_links: '-1'
        odbc.defaultlrl: 4096
        odbc.defaultbinmode: 1
        mysql.allow_persistent: On
        mysql.max_persistent: '-1'
        mysql.max_links: '-1'
        mysql.connect_timeout: 60
        mysql.trace_mode: Off
        mysqli.max_links: '-1'
        mysqli.default_port: 3306
        mysqli.reconnect: Off
        pgsql.allow_persistent: On
        pgsql.auto_reset_persistent: Off
        pgsql.max_persistent: '-1'
        pgsql.max_links: '-1'
        pgsql.ignore_notice: 0
        pgsql.log_notice: 0
        sybct.allow_persistent: On
        sybct.max_persistent: '-1'
        sybct.max_links: '-1'
        sybct.min_server_severity: 10
        sybct.min_client_severity: 10
        bcmath.scale: 0
        session.save_handler: 'files'
        session.use_cookies: 1
        session.use_only_cookies: 1
        session.name: 'PHPSESSID'
        session.auto_start: 0
        session.cookie_lifetime: 0
        session.cookie_path: /
        session.serialize_handler: php
        session.gc_probability: 1
        session.gc_divisor: 1000
        session.gc_maxlifetime: 1440
        session.bug_compat_42: Off
        session.bug_compat_warn: Off
        session.entropy_length: 0
        session.cache_limiter: nocache
        session.cache_expire: 180
        session.use_trans_sid: 0
        session.hash_function: 0
        session.hash_bits_per_character: 5
        url_rewriter.tags: "a=href,area=href,frame=src,input=src,form=fakeentry"
        mssql.allow_persistent: On
        mssql.max_persistent: '-1'
        mssql.max_links: '-1'
        mssql.min_error_severity: 10
        mssql.min_message_severity: 10
        mssql.compatability_mode: Off
        mssql.secure_connection: Off
        tidy.clean_output: Off
        soap.wsdl_cache_enabled: 1
        soap.wsdl_cache_dir: "/tmp"
        soap.wsdl_cache_ttl: 86400
        mod_php: 0

```

highlighting date.timezone, this is not represented in the php.ini file itself.

```

$ cat php.ini | grep "timezone"
867:; Defines the default timezone used by the date functions
868:; http://php.net/date.timezone
869:;date.timezone =

```

It has remained commented.

Am I missing something?

**Comment by jtreminio** [comment]
The PHP timezone setting is different due to a legacy issue that I can most likely clean up now.

    php:
        install: '1'
        version: '56'
        composer: '1'
        composer_home: ''
        modules:
            php:
                - cli
                - intl
                - mcrypt
            pear: {  }
            pecl:
                - pecl_http
        ini:
            display_errors: On
            error_reporting: '-1'
            session.save_path: /var/lib/php/session
        timezone: America/Chicago
        mod_php: 0

You want to set it at `php.timezone`

I'll implement a change to this soonish.

**Comment by timmigrant** [comment]
Hi,

Thank you for the help, I found out why values are not being set.

it is this line:

```
url_rewriter.tags: "a=href,area=href,frame=src,input=src,form=fakeentry"
```

is being translated too:

```
url_rewriter.tags=a=href,area=href,frame=src,input=src,form=fakeentry
```

and it was throwing an error.

I temporarily removed that line and everything was set properly.

**Comment by jtreminio** [comment]
Closed by 7160b2ecbe995f5d12225aa75a7dd168c76b958c
